crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -393,14 +393,24 @@ def proposal_update
           if answer[0] == 202
             missing_nodes = answer[1].map { |node_dns| NodeObject.find_node_by_name(node_dns) }
 
-            unready_nodes = missing_nodes.map(&:alias)
+            unready_nodes = missing_nodes.select { |n| n.state != "ready" }.map(&:alias)
             unallocated_nodes = missing_nodes.reject(&:allocated?).map(&:alias)
 
-            flash[:notice] = "#{t('barclamp.proposal_show.commit_proposal_queued')}: #{(unready_nodes - unallocated_nodes).join(", ")}"
-
+            unless unready_nodes.empty?
+              flash[:notice] = "#{t('barclamp.proposal_show.commit_proposal_queued')}: #{(unready_nodes - unallocated_nodes).join(", ")}"
```

Comment:
PLEASE use the power of rails i18n:

```ruby
t(
  "barclamp.proposal_show.commit_proposal_queued",
  nodes: (unready_nodes - unallocated_nodes).join(", ")
)
```

```yaml
commit_proposal_queued_dependency: 'Successfully queued the proposal until %{nodes} are deployed'
```

That should be applied for all the translation strings. Do you know what happens if we add more languages that require the list on a different place?